### DISCUSSION TITLE
VS Code Terminal Black Box

### DISCUSSION THREAD

**Discussion by pointdexture** [Cyfrin/foundry-full-course-cu#485] [General]
Hii,
I am getting this BLACK BOX in my terminal ( I attached a screenshot of vs code black box ). The problem is terminal commands are not visible in the Black Box area. I uninstall the VScode and also delete the .vscode and Code file from ->program file & ->app data and then reinstall it. The problem remains the same. 

Please help to guide what to do, I will be very thankful.

![vscodeerror](https://github.com/Cyfrin/foundry-full-course-f23/assets/112416293/0239d5a1-9fa7-4642-b1bd-305bddf4d6f8)

**Comment by ChefAharoni** [comment]
Try the fixes listed [here](https://github.com/Microsoft/vscode/issues/49488).
